saltstack/optional/salt/oracle-java/init.sls

```diff
@@ -13,46 +13,33 @@ remove_openjdk:
       - java-1.8.0-openjdk-devel
       - java-1.8.0-openjdk-javadoc
       - java-1.8.0-openjdk-src
-    {% elif grains['os_family'] == 'Debian' %}
-      - openjdk-7-jre-headless
-      - openjdk-7-doc
-      - openjdk-7-source
-      - openjdk-7-jdk
-      - openjdk-8-jre-headless
-      - openjdk-8-doc
-      - openjdk-8-source
-      - openjdk-8-jdk
-    {% endif %}
 
-
-{% if grains['os_family'] == 'RedHat' %}
 download_oracle_jdk:
   {% set jdk_url = salt['environ.get']('ORACLE_JDK8_URL_RPM') %}
-  {% set download_jdk_cmd = ['\'wget --no-cookies --no-check-certificate --header "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" "',
+  {% set download_jdk_cmd = ['\'wget -c -q --no-cookies --no-check-certificate --header "Cookie: gpw_e24=http%3A%2F%2Fwww.oracle.com%2F; oraclelicense=accept-securebackup-cookie" "',
                              jdk_url,
                              '" -O /tmp/oracle_jdk_install.rpm\'' ] %}
   cmd.run:
     - name: {{ download_jdk_cmd | join('') }}
-    - unless: test -e /tmp/oracle_jdk_install.rpm
```

Comment:
With this option it was possible to place an rpm there and install it without downloading. Is it really necessary to remove?

Reply:
With the -c option of wget we get the same functionality.